### ISSUE TITLE
Pre-trained model weights

### ISSUE THREAD

**Issue by GODGANG4885** [zcemycl/Pytorch_Outpainting_SRN#1]
Thanks for sharing this awesome re-implementation.
Can you please provide pre-trained model weights so that we can finetune your model?

**Comment by zcemycl** [comment]
@GODGANG4885

Sure. Can you leave me your email address?

I am not certain if this is a good implementation, so the weights might not be pretrained as expected. To clarify, you want the pretrained weights from the first step (pretrain wgan)? Or the second step (finetune srn)?

I trained the model in the following settings due to my limited computation resources, 
1. 
```
python main.py --pretrained_network=1 --batch_size=16 --gan_loss_alpha=0 --pretrain_l1_alpha=5 --mrf_alpha=0 --lrG=1e-5 --lrD=5e-5 --beta1 0.5 --beta2 0.9 --summarydir=log/store
```
2.
 ```python main.py --pretrained_network=0 --weightpath=log/store --batch_size=8 
--lrG=1e-5 --lrD=5e-5 --mrf_alpha=0.05 --summarydir=log/store5 --beta1=.5 --beta2=.9 --pretrain_l1_alpha=1.2  --gan_loss_alpha=0.001 
```

Refer to the paper, 

> To better stabilize the adversarial training, our model ispre-trained  first  with  only  reconstruction  loss  (λs= 5).Afterwards,  we  letλmrf= 0.05andλadv= 0.001forfine-tuning SRN until convergence. During training, Adamsolver  [22]  with  learning  rate1e−4is  adopted  whereβ1= 0.5andβ2= 0.9.   Training batch size is 16.   Theinput and output are linearly scaled within range[−1,1]

**Comment by kareem337** [comment]
Thank for this great work. i have trained the model succefully but im not sure how to test the model or how to put the validator dataset to test the model. plus where can i find the output. thanks in advance!!

**Comment by zcemycl** [comment]
@kareem337 The code I had written does not contain a test/deploy script. To test it with another dataset, i would recommend to read `data.py`, it shows how to load images into dataloader. Then, in `main.py`, there is a line to call the loader, please replace it with another dataloader. Finally, evaluate the model to generate the outerpolated image and save it with typical matplotlib method. 

This repo is a trial version for me to replicate the paper, the performance is not guaranteed to be good.

**Comment by kareem337** [comment]
It worked! Thank you for your help. Since you are saying that the code is not guaranteed to be good, what kind of methods or files are missing so that the code can outpaint an image successfully.
